src/azure-cli/azure/cli/command_modules/appservice/static_sites.py

```diff
@@ -520,7 +520,7 @@ def delete_staticsite(cmd, name, resource_group_name=None, no_wait=False):
 
 def _parse_pair(pair, delimiter):
     if delimiter not in pair:
-        InvalidArgumentValueError("invalid format of pair {0}".format(pair))
+        raise InvalidArgumentValueError("invalid format of pair {0}".format(pair))
```

Comment:
This line makes no sense without `raise`.
Detected by `pointless-exception-statement` rule.